app/lib/paymentUtils.js

```diff
@@ -30,8 +31,45 @@ function hasTemporaryPremiumAccess() {
   return storage.load(`${TEMPORARY_PREMIUM_KEY}-${me.get('_id')}`)
 }
 
+function setTrackedPremiumPurchase() {
+  storage.save(`${TEMPORARY_PREMIUM_KEY}-${me.get('_id')}`, true, 6 * 60)
+}
+
+function hasTrackedPremiumAccess() {
+  return storage.load(`${TRACKED_PREMIUM}-${me.get('_id')}`)
+}
+
+function getTrackingData ({ amount, duration }) {
+  const options = {}
+  if (amount) {
+    const numericalAmount = parseInt(amount) / 100
+    options.value = `${numericalAmount}`
+    options.currency = 'USD'
+    // ltv -> calculate based on duration
+    options.predictedLtv = `${numericalAmount} * ${getLtvMultiplier(duration)}`
+  }
+  return options
+}
+
+function getLtvMultiplier (duration) {
+  if (!duration)
+    return 1
+  if (duration.includes('year'))
+    return 1
```

Comment:
That would be akin to saying a 0% renewal rate. As a super rough approximation, probably 2 would be better (renews once on average).